float64 alpha
float64 beta
float64 gamma
float64[] delta_x_d
float64[] J
float64[] desired_force_decrease
float64[] x_0
float64[] KP_t_KP
float64[] B
float64[] q_min
float64[] q_max
float64[] f_min
float64[] f_max
float64[] n_K_ci_J_ci
float64[] n_K_ci_J_ci_max
float64[] u_min
float64[] u_max
---
float64[3] delta_phi_opt
float64[3] predicted_joint_angles




